community/kernel/src/main/java/org/neo4j/kernel/impl/store/DumpStore.java

```diff
@@ -102,7 +102,7 @@ else if ( !file.isDirectory() && file.getName().indexOf( ':' ) != -1 )
                 throw new IllegalArgumentException( "No such file: " + arg );
             }
         }
-        try ( NeoStores neoStores = createStoreFactory.apply( file ).openNeoStoresEagerly() )
+        try ( NeoStores neoStores = createStoreFactory.apply( file ).openNeoStores() )
```

Comment:
It's desirable to make tools like this work on incomplete stores. I think it's probably a mistake that this line has been using eager store opening. Perhaps we can make it map the store file names to `StoreType` enum values, and use that to only open and work with the specific store we're interested in?

Same deal for `DumpStoreChain`.